.github/workflows/ci.yaml

```diff
@@ -179,7 +187,7 @@ jobs:
 
     - name: Publish to PyPI - on tag
       if: startsWith(github.ref, 'refs/tags/')
-      uses: pypa/gh-action-pypi-publish@a56da0b891b3dc519c7ee3284aff1fad93cc8598
+      uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I went with the automatically updated version here.

Since constantly is not that active, I think is ok to pin to the tag